packages/react/src/alert-dialog/AlertDialog.css.ts

```diff
@@ -63,8 +63,8 @@ export const footer = recipe({
 export const title = recipe({
   base: [
     {
-      fontSize: "2xl",
-      fontWeight: "600",
+      fontSize: "3xl",
+      fontWeight: "500",
```

Comment:
can we re-use the `Heading` component instead of setting the `fontSize` directly?

so in the `AlertDialog.tsx` file we should have something like:

```tsx
<Heading asChild fontWeight="600" level="3" p="24" pb="16">
  <RadixAlertDialog.Title>{title}</RadixAlertDialog.Title>
</Heading>
```